modules/troubleshooting-common-problems-with-distributed-workloads-for-users.adoc

```diff
@@ -0,0 +1,234 @@
+. Resolve the problem in one of the following ways:
+
+* If a local queue exists, ensure that you spelled the local queue name correctly in your cluster configuration, and that the `namespace` value in the cluster configuration matches your project name.
+If you do not specify a `namespace` value in the cluster configuration, the Ray cluster is created in the current project.
+
+* If no local queue exists, contact your administrator to request assistance.
+
+== I cannot create a Ray cluster or submit jobs
+
+.Problem
+After you run the `cluster.up()` command, an error similar to the following error is shown:
+
+[source,bash]
+----
+RuntimeError: Failed to get RayCluster CustomResourceDefinition: (403)
+Reason: Forbidden
+HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"rayclusters.ray.io is forbidden: User \"system:serviceaccount:regularuser-project:regularuser-workbench\" cannot list resource \"rayclusters\" in API group \"ray.io\" in the namespace \"regularuser-project\"","reason":"Forbidden","details":{"group":"ray.io","kind":"rayclusters"},"code":403}
+----
+
+.Diagnosis
+The correct OpenShift login credentials are not specified in the `TokenAuthentication` section of your notebook code.
+
+.Resolution
+. Identify the correct OpenShift login credentials as follows:
+
+ifdef::upstream,self-managed[]
+.. In the {openshift-platform} console header, click your username and click *Copy login command*.
+endif::[]
+ifdef::cloud-service[]
+.. In the OpenShift console header, click your username and click *Copy login command*.
+endif::[]
+
+.. In the new tab that opens, log in as the user whose credentials you want to use.
+.. Click *Display Token*.
+.. From the *Log in with this token* section, copy the `token` and `server` values.
+
+. In your notebook code, specify the copied `token` and `server` values as follows:
++
+[source,bash,subs="+quotes"]
+----
+auth = TokenAuthentication(
+    token = "_<token>_",
+    server = "_<server>_",
+    skip_tls=False
+)
+auth.login()
+----
+
+
+== Kueue does not monitor my Ray cluster or PyTorch job
+
+.Problem
+Kueue does not monitor the Ray cluster, or PyTorch jobs, or other machine-learning frameworks.
+
+.Diagnosis
+The resources (for example,`RayCluster` resources and `PyTorchJob` resources) are not created.
+
+.Resolution
+Contact your administrator to request assistance.
+
+== My pod provisioned by Kueue is terminated before my image is pulled
+
+.Problem
+Kueue waits for a period of time before marking a workload as ready, to enable all of the workload pods to become provisioned and running. 
+By default, Kueue waits for 5 minutes. 
+If the pod image is very large and is still being pulled after the 5-minute waiting period elapses, Kueue fails the workload and terminates the related pods.
+
+.Diagnosis
+Review the Events of the Ray head pod to check whether the image pull completed successfully.
```

Comment:
does "Events of the Ray" need any formatting applied? maybe bold text?

Reply:
Rephrased to clarify:
"Open the *Events* tab for the Ray head pod, and review the pod events to check  ..."